internal/powerpath/generate_mock.sh

```diff
@@ -1,3 +1,15 @@
+# Copyright © 2020 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
It should be 2022 As we created this file in 2022.

Reply:
changed it

Reply:
Thank you.